src/App.jsx

```diff
@@ -2,6 +2,7 @@ import { createSignal, createEffect, onCleanup } from 'solid-js';
 import NavBar from './components/NavBar';
 import MainContainer from './containers/MainContainer';
 import Footer from './components/Footer';
+import Error404 from './Error404';
```

Comment:
Do we need this import here in App.jsx?

Reply:
Yes, you are right. Thanks for the feedback!